nplbam/app/templates/edit_account.html

```diff
@@ -21,10 +21,10 @@
             </div>
             <div class="item">
                 <label for="userLVL">User Access Level: </label>
+                <input type="number" id="userLVL" value="{{ info.get("user_lvl") }}" name="userLVL" />
```

Comment:
Didn't you move this to the bottom before?

Reply:
or was that from a different file?

Reply:
That was the new_account page this change is to match it